src/ott/datasets.py

```diff
@@ -110,7 +110,7 @@ def create_gaussian_mixture_samplers(
     name_target: Name_t,
     train_batch_size: int = 2048,
     valid_batch_size: int = 2048,
-    rng: Optional[jax.Array] = None,
+    rng: Optional[jnp.ndarray] = None,
```

Comment:
I think for the rng we do need to pass them as `jax.Array`, @michalk8  can you confirm?